app/views/govuk_publishing_components/components/_single_page_notification_button.html.erb

```diff
@@ -10,6 +10,9 @@
   wrapper_classes = %w(gem-c-single-page-notification-button govuk-!-display-none-print)
   wrapper_classes << shared_helper.get_margin_bottom
   classes = "govuk-body-s gem-c-single-page-notification-button__submit"
+  
+  data_attributes[:module] = "single-page-notification-button"
```

Comment:
Can we make this overridable by an app?  We can serve two versions of a page: one for users with a cookie, one for users without a cookie.  Only the version for users with a cookie needs to run the javascript.